extensions/cornerstone-dicom-seg/src/getSopClassHandlerModule.ts

```diff
@@ -93,8 +93,19 @@ function _getDisplaySetsFromSeries(
     return referencedDisplaySet;
   };
 
-  displaySet.load = async ({ headers }) =>
-    await _load(displaySet, servicesManager, extensionManager, headers);
+  displaySet.load = async ({ headers }) => {
```

Comment:
This probably does not belong here, can you put it in the cacheService that calls this? and make it general?

Reply:
Done!